tensorflow/lite/delegates/gpu/BUILD

```diff
@@ -141,7 +141,7 @@ cc_binary(
     deps = [":gl_delegate"],
 )
 
-# build -c opt --config android_arm64 --copt -Os --copt -DTFLITE_GPU_BINARY_RELEASE --copt --linkopt -s --strip always :libtensorflowlite_gpu_delegate.so
+# build -c opt --config android_arm64 --copt -Os --copt -DTFLITE_GPU_BINARY_RELEASE --copt -s --strip always :libtensorflowlite_gpu_delegate.so
```

Comment:
Same with this line, sorry should have flagged earlier.

Reply:
ah that is my bad--I haven't looked at this file in a while. Fixed in https://github.com/tensorflow/tensorflow/pull/45049/commits/4914abba00cb5b278d9cf3b7ef0c2345357a72ed